dc-model/src/main/java/de/digitalcollections/model/filter/Filtering.java

```diff
@@ -66,4 +67,26 @@ public FilterCriterion getFilterCriterionFor(String property) {
   public void setFilterCriteria(List<FilterCriterion> filterCriteria) {
     this.filterCriteria = filterCriteria;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof Filtering)) {
+      return false;
+    }
+    Filtering filtering = (Filtering) o;
+    return Objects.equals(filterCriteria, filtering.filterCriteria);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(filterCriteria);
```

Comment:
Hier sollte noch irgendwas dazu, damit nicht die Liste identisch mit diesem `Filtering`-Objekt ist. Vorschlag:
```suggestion
    return Objects.hash(filterCriteria) + Objects.hash("Filtering");
```

Reply:
ist eingebaut